protoc-gen-openapiv2/internal/genopenapi/BUILD.bazel

```diff
@@ -24,6 +24,7 @@ go_library(
         "@go_googleapis//google/api:annotations_go_proto",
         "@go_googleapis//google/rpc:status_go_proto",
         "@io_bazel_rules_go//proto/wkt:any_go_proto",
+        "@org_golang_google_genproto//googleapis/api/visibility",
```

Comment:
These references also look wrong compared to the `annotations_go_proto` on line 24